### PR TITLE
more templates

### PR DESCRIPTION
**Pull request by Julian-Chu**
Hi @philipgriggs ,
Thank you so much for this go-postfix-templates. It's very useful in my daily coding life with Goland.
Here I also have some own postfix templates, would like to integrate to your repository.   Could you review the pull request?

### CODE REVIEW

**Comment by philipgriggs** [comment]
Thanks for the contribution and your feedback 👍 I'd like to include more templates in the repo, so please create another PR if you have any more ideas that would be useful.

File: http.postifxTemplates
Comment (philipgriggs) [inline http.postifxTemplates]: This example would work as a `GoLand` live template (https://www.jetbrains.com/help/go/using-live-templates.html). Do you have a good reason for including it in `custom-postfix-templates` instead? I'd consider adding it to `custom-postfix-templates` if there is an advantage.

Let me know if you're not familiar with how the live templates work.

File: http.postifxTemplates
Comment (philipgriggs) [inline http.postifxTemplates]: same as above

File: testing.postfixTemplates
Comment (philipgriggs) [inline testing.postfixTemplates]: there is already a live template for `test` (eg If you type `test` somewhere in the editor). Again, is there an advantage to include it here as well?

File: testing.postfixTemplates
Comment (philipgriggs) [inline testing.postfixTemplates]: as above, there is already a live template for `bench`

File: http.postifxTemplates
Comment (Julian-Chu) [reply http.postifxTemplates]: @philipgriggs , 

Thanks for your comments. I also have  some live templates with postfix templates format for my coding habit. But I think you opinion make sense to keep clear postfix templates. I delete them from this repository,  only just keep in my user-defined postfix.

File: general.postfixTemplates
Comment (philipgriggs) [inline general.postfixTemplates]: can you move these to `misc.postfixTemplates`? My idea was to include general templates in that file 
https://github.com/philipgriggs/go-postfix-templates/blob/master/misc.postfixTemplates

**Review by philipgriggs — CHANGES_REQUESTED** [review]

File: general.postfixTemplates
Comment (Julian-Chu) [reply general.postfixTemplates]: done!

**Review by philipgriggs — APPROVED** [review]

**Comment by Julian-Chu** [comment]
hi @philipgriggs ,  just reminding,  I don't have write access to this repo, so can't merge it by myself.

**Pull request merged**
